custom/enikshay/integrations/bets/const.py

```diff
@@ -1,3 +1,20 @@
+# BETS event names
+VOUCHER_EVENT_ID = "VOUCHER_EVENT_ID"  # TODO: Get this from BETS
+TREATMENT_180_EVENT = "104"
+DRUG_REFILL_EVENT = "109"
+SUCCESSFUL_TREATMENT_EVENT = "SUCCESSFUL_TREATMENT_EVENT"  # TODO: Get this from BETS
+DIAGNOSIS_AND_NOTIFICATION_EVENT = "103"
+AYUSH_REFERRAL_EVENT = "AYUSH_REFERRAL_EVENT"  # TODO: Get this from BETS
```

Comment:
The `BETS_EVENT_IDS` dict below contains all the event IDs they've sent over.  `VOUCHER_EVENT_ID` is actually two IDs, I think - `chemist_voucher` and `lab_voucher` below.

`AYUSH_REFERRAL_EVENT` corresponds to event 105 below
>  AYUSH/Other provider -Registering and referral of a presumptive TB case in UATBC/e-Nikshay, and patient subsequently gets notified

I don't see an event corresponding to `SUCCESSFUL_TREATMENT_EVENT`, which I believe is this in long form:
> Patients: Cash transfer on successful treatment completion (* trigger - updating of ‘Treatment Outcome’ for an episode )

I'll ping about that one.

Also, the keys in the dict below aren't actually being used for anything - for my part of this I only need a list of valid event IDs.  I stuck the slugs in there as a way to assign names to them.  If you prefer to use individual constants, then we should get rid of that dict and just have a list of the constants so we don't have competing names.